nmbrs_hibob.py

```diff
@@ -262,21 +258,37 @@ def fetch_salary_slips(year, run):
                 employee_response = do_request(get_employee(employee.id), emp)
                 employee_details = to_employee_details(employee_response)
 
-                request_body = get_payslip(employee.id, run_arg, year)
+                request_body = get_payslip(employee.id, Arguments.run_arg, year)
                 response = do_request(request_body, com)
                 pdfs = response.findall(f'.//cs:PDF', namespaces=ns)
 
                 for index, pdf in enumerate(pdfs):
-                    description = run_info.description.replace(' ', '_') if description_arg is None else description_arg
-                    folder_name = employee_details.email if email_arg else employee_details.number
+                    description = run_info.description.replace(' ', '_') if Arguments.description_arg is None \
+                        else Arguments.description_arg
+                    folder_name = employee_details.email if Arguments.email_arg else employee_details.number
                     file_name = f"{folder_name}/{year}_{run_info.number}_{description}_{index}.pdf"
                     zip_file.writestr(file_name, base64.b64decode(pdf.text))
 
                 bar.next()
         spinner.succeed(f"Wrote payslips to {zip_file_name}")
 
 
-if annual_arg:
-    fetch_annual_statements(year_arg)
-else:
-    fetch_salary_slips(year_arg, run_arg)
+if __name__ == '__main__':
+    spinner = Halo(text='Determining company ID..', spinner='dots')
```

Comment:
In the monorepo I've replaced Halo with the much more comprehensive `rich` library https://github.com/Textualize/rich.